### ISSUE TITLE
`ros-cdk version` fails with `Cannot find module uuid`

### ISSUE THREAD

**Issue by wojtekwalczak** [aliyun/Resource-Orchestration-Service-Cloud-Development-Kit#23]
Hi,

there are some dependencies missing for newest version of `ros-cdk` (1.0.12).

You can replicate the problem in the following way (I run it with docker to make sure that the problem is not located in my local environment):

```bash
$ docker run -it node:latest /bin/bash
...
# npm install lerna -g
...
# npm install @alicloud/ros-cdk-cli@1.0.12 -g
...
# ros-cdk version
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module 'uuid'
Require stack:
- /usr/local/lib/node_modules/@alicloud/ros-cdk-cli/lib/cdk-toolkit.js
- /usr/local/lib/node_modules/@alicloud/ros-cdk-cli/bin/cdk.js
- /usr/local/lib/node_modules/@alicloud/ros-cdk-cli/bin/ros-cdk
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/@alicloud/ros-cdk-cli/lib/cdk-toolkit.js:23:16)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/local/lib/node_modules/@alicloud/ros-cdk-cli/lib/cdk-toolkit.js',
    '/usr/local/lib/node_modules/@alicloud/ros-cdk-cli/bin/cdk.js',
    '/usr/local/lib/node_modules/@alicloud/ros-cdk-cli/bin/ros-cdk'
  ]
}
```

**Comment by Ehzyo** [comment]
We will fix the issue as soon as possible.

**Comment by Ehzyo** [comment]
We have fix the issue, and ros-cdk-cli（1.0.13) is released.
